tests/hardware_reboot/testdata/hardware_audio_reboot.txt

```diff
@@ -0,0 +1,102 @@
+# Simple test of Audio device after reboot of guest
+
+{{define "port"}}2223{{end}}
+{{define "ssh"}} ssh -oServerAliveInterval=10 -oConnectTimeout=10 -oStrictHostKeyChecking=no -oPasswordAuthentication=no -i {{EdenConfig "eden.tests"}}/eclient/image/cert/id_rsa -p {{template "port"}} root@$HOST {{end}}
+
+[!exec:bash] stop
+[!exec:sleep] stop
+[!exec:ssh] stop
+
+# Starting of reboot detector with a 2 reboot limit
+! test eden.reboot.test -test.v -timewait 10m -reboot=0 -count=2 &
+
+eden pod deploy -n eclient --memory=512MB docker://itmoeve/eclient:0.7 -p {{template "port"}}:22
+
+test eden.app.test -test.v -timewait 20m RUNNING eclient
+
+exec -t 20m bash ssh.sh {{template "port"}}
+stdout 'Ubuntu'
+
+# Audio device passthrough setup
+exec -t 20m bash set_audio.sh eclient
+stdout 'msg="Config loaded"'
+
+# Wait for reconfig
+exec -t 20m bash ssh.sh
+stdout 'Ubuntu'
+
```

Comment:
I think, you should add here `exec -t 2m bash get-lshw.sh {{template "port"}}` to check output before reboot.